Get user session guide (#3886)
Nexmo/nexmo-developer: _documentation/en/client-sdk/in-app-messaging/guides/get-user-sessions.md

---
title: Get User Sessions
navigation_weight: 10
---

# Get User Sessions

## Overview

> This guide covers how to get the a user's active [sessions](/conversation/concepts/session), which can be used as a way to show an online status. 

Reviewer: ```suggestion
This guide covers how to get a user's active [sessions](/conversation/concepts/session), which can be used as a way to show their online status. 
```

> Before you begin, make sure you [added the SDK to your app](/client-sdk/setup/add-sdk-to-your-app) and you are able to [create a conversation](/client-sdk/in-app-messaging/guides/simple-conversation).

Reviewer: ```suggestion
Before you begin, make sure you [added the SDK to your app](/client-sdk/setup/add-sdk-to-your-app) and are able to [create a conversation](/client-sdk/in-app-messaging/guides/simple-conversation).
```
